app/src/main/java/com/mk/tasky/core/presentation/MainActivity.kt

```diff
@@ -203,6 +205,12 @@ fun MainScreen(
         ) {
             val eventTitle = it.savedStateHandle.get<String>("event_title") ?: ""
             val eventDescription = it.savedStateHandle.get<String>("event_description") ?: ""
+            val selectedPhoto = it.savedStateHandle.get<String>("photo_location") ?: ""
+            val deletePhoto = it.savedStateHandle.get<Boolean>("should_delete") ?: false
+
+            val deletablePhoto =
+                if (deletePhoto && selectedPhoto.isNotBlank()) selectedPhoto else null
```

Comment:
Also here you could remove the second part of the if-statement if `selectedPhoto` would be nullable

Reply:
Replaced the IF